### ISSUE TITLE
The total pagenumber on the toolbar is elliptical

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#109]
```
What steps will reproduce the problem?


What is the expected output? What do you see instead?
it's best show the total pagenumber completely

What version of the product are you using? On what operating system?
the src code of the latest

Please provide any additional information below.

```

Original issue reported on code.google.com by `yiditu...@163.com` on 19 Aug 2010 at 5:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `yiditu...@163.com` on 19 Aug 2010 at 5:29

Attachments:
- [ccdd.JPG](https://storage.googleapis.com/google-code-attachments/flexpaper/issue-109/comment-1/ccdd.JPG)

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.4.0 rc2
```

Original comment by `erik.eng...@gmail.com` on 19 Aug 2010 at 4:11
- Changed state: **Fixed**
